data/storage/postgres/create_db_tables.sql

```diff
@@ -152,3 +152,46 @@ CREATE TABLE route_profiles (
   UNIQUE (tenant, id)
 );
 CREATE UNIQUE INDEX route_profiles_idx ON route_profiles ("id");
+
+
+DROP TABLE IF EXISTS rates;
+DROP TABLE IF EXISTS rate_profiles;
+CREATE TABLE rate_profiles (
+  pk SERIAL PRIMARY KEY,
+  tenant VARCHAR(40) NOT NULL,
+  id VARCHAR(64) NOT NULL,
+  rate_profile JSONB NOT NULL,
+  UNIQUE (tenant, id)
+);
+CREATE UNIQUE INDEX rate_profiles_idx ON rate_profiles ("id");
+CREATE TABLE rates (
+  pk SERIAL PRIMARY KEY,
+  tenant VARCHAR(40) NOT NULL,
+  id VARCHAR(64) NOT NULL,
+  rate JSONB NOT NULL,
+  rate_profile_id VARCHAR(64) NOT NULL,
```

Comment:
Please reverse order here at schema definition, keep rate_profile_id before id of rate, so we can follow the format of struct.